safaribooks.py

```diff
@@ -1121,4 +1121,4 @@ def create_epub(self):
 
     SafariBooks(args_parsed)
     # Hint: do you want to download more then one book once, initialized more than one instance of `SafariBooks`...
-    sys.exit(0)
+    sys.exit(0)
```

Comment:
**Suggestion:** Avoid using sys.exit(0) directly in the code. It's better to return from the function or raise an exception that can be caught at a higher level. This will make the code more maintainable and easier to test.
```suggestion
    SafariBooks(args_parsed)
    # Hint: do you want to download more then one book once, initialized more than one instance of `SafariBooks`...
    return
```